.babelrc

```diff
@@ -20,11 +20,12 @@
     "test": {
       "plugins": [
         ["@babel/plugin-proposal-class-properties", { "loose": true }],
-        "@babel/plugin-transform-runtime",
+        "@babel/plugin-transform-runtime"
       ]
     },
     "production": {
       "plugins": [
+        "@babel/plugin-proposal-class-properties",
```

Comment:
@ayodejiAA Do we need this plugin in production? I believe our app didn't break on Heroku when we didn't add the plugin. Could you clarify, please?

Reply:
Build failed when it was not added. It is necessary we have it so we can use state in components without having a constructor.

Reply:
Alright then.